src/main/java/unrc/dose/User.java

```diff
@@ -1,19 +1,39 @@
-/*
-* == Schema Info
-*
-* Table name: users
-*
-*  id              :integer(11)    not null, primary key
-*  username        :varchar(20)    not null,
-*  password        :varchar(20)    not null,
-*  email_address   :varchar(50)    not null,
-*  admin           BOOLEAN not null default 0
-*
-**/
```

Comment:
Why did you remove these comments? These should be present to describe the associated table